src/Taonomics.md

```diff
@@ -6,54 +6,35 @@ Tao is the token for Bittensor and gatekeeps access, representing bandwidth to t
 
 
 
-# Emission Split
+**Emission Split**: Each subnetwork \\(h\\) receives a set percentage \\(E_h\\) of total emission \\(E=\sum_h E_h\\).
+Servers and Validators evenly split emissions per block, with the most emissions issued to the best-performing Servers and largest Validators.
 
-Servers and Validators evenly split emissions per block, with the most emissions issued to the best-performing Servers.
+**Weights**: Each Validator \\(i\\) evaluates the relative utility of each Server \\(j\\), and sets weights \\(\sum_jW_{ij}=1\\) on-chain, signed with Validator hotkey. A relatively high weight indicates relatively high Server utility.
 
+**Prerank**: Server rank \\(R_j\\) is the sum of weighted stake set on \\(j\\), before consensus. A high rank indicates that majority stake agrees on relatively high utility of a Server.
+\\[P_j = \sum_i S_i \cdot W_{ij}\\]
 
-# Validation and B (Bonds)
+**Consensus**: On-chain consensus then calculates stake-based median weight \\(C_j\\) for each Server \\(j\\), which is the maximum weight supported by at least \\(\kappa\\)-majority stake, so that stake sum (as ratio of total active stake) of active Validators setting at least consensus weight is \\(\kappa\\) (typically \\(\kappa=0.5\\)).
+\\[C_j = \max w:\quad \kappa \le \sum_i S_i \cdot \left( w \le W_{ij} \right)\\]
+<img src="images/consensus_plots.pdf"  height="200">
 
-The Validators determine the proportion of emissions received by each Server in the network. Validators, responsible for validating the action of Servers, assign rankings to Servers based on their performance. In doing so, Validators accumulate bonded relationships in the Servers they rank. Unlike traditional bonding, our bonding is defined by:
+**Validator Trust**: Clip weights above consensus to penalize overweighting, the sum of consensus-clipped weight is then Validator trust \\(V_i\\), which is the weighting power remaining after consensus (initially starts with 100% power). High Validator trust means it is typically setting weights in agreement with majority stake.
+\\[V_i = \sum_j\max\left( W_{ij}, C_j \right)\\]
 
-\\[ {\Delta}{B}={W}*{S} \\]
+**Server Incentive**: Server rank \\(R_j\\) is the sum of weighted stake set on \\(j\\), which is also the Server incentive ratio \\(I_j\\). Better performing Servers get relatively higher rank and incentive.
+\\[I_j = R_j = \sum_i S_i \cdot \max\left( W_{ij}, C_j \right)\\]
 
-In this way, peers accumulate bonds in the peers they rank, thus ’bonding’ themselves to those that they are connected to.
+**Server Trust**: The ratio of Server \\(j\\) rank \\(R_j\\) to its prerank \\(P_j\\) is the trust \\(T_j\\) in the initial weights set on the Server. High Server trust means that mostly trusted Validators set weights on it.
```

Comment:
I recommend replacing "means" when not describing averages with a synonym such as "dictates" or "demonstrates" to avoid convolution.